src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To-Do List</title>
+</head>
+<body>
+  <div>
+    <ul>
+      <li><span>Today's To Do</span><img class="sync" src="7a51210ee31f44c8cd5f.png" alt="sync" id="sync"></li>
+      <li><input class="InputText" type="text" placeholder="Add to your list..."><img class="enter" src="23e71aaf97dc09e15144.png" alt="enter" id="enter"></li>
+    </ul>
```

Comment:
Here, you put inside the Unorder list the title and the input as two more elements. It is a very ingenious and efficient solution but I think it would be more appropriate if they were outside since they are not elements of the UL.